src/app/api/snippets/route.ts

```
@@ -0,0 +1,20 @@
+import { NextResponse } from "next/server";
+import fs from "fs";
+import path from "path";
+
+export async function GET() {
+  const snippetsPath = path.join(process.cwd(), "consolidated");
+  const files = fs
+    .readdirSync(snippetsPath)
+    .filter((file) => file !== "_relationships.json"); // Exclude relationships.json;
+
+  let allSnippets: any[] = [];
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)